### DISCUSSION TITLE
multi modal binary classification

### DISCUSSION THREAD

**Discussion by irazza** [alan-turing-institute/sktime#4787] [Q&A]
Hi,
I'm currently working on a multi-modal binary classification problem and would greatly appreciate some guidance and assistance. I have two input signals for every observation and need to predict a single binary label as the output.

I've already explored the documentation and examples provided by `sktime`, but I'm unsure about the specific steps and techniques to use for this particular scenario. I'm reaching out to the community to seek advice and suggestions on how to approach this problem effectively using `sktime`.

Here are some specific questions I have:

1. How can I represent the multi-modal input signals in the required format for `sktime`? Should I reshape my data manually or are there specific functions available for this purpose?

2. Which classifier in `sktime` would be suitable for this multi-modal binary classification problem? Are there any specific considerations or parameters that I should be aware of?

3. What are the best practices or recommended workflows for preprocessing and feature extraction in a multi-modal context using `sktime`?

I would greatly appreciate any insights, code snippets, or references to relevant resources that can help me tackle this problem effectively. If you have previously worked on similar multi-modal classification tasks with `sktime`, I would love to hear about your experiences and any lessons learned.

Thank you in advance for your time and assistance. I'm looking forward to your valuable input and guidance!

**Comment by fkiraly** [comment]
closing (likely accidental) duplicate of https://github.com/sktime/sktime/discussions/4786
